### DISCUSSION TITLE
Inference API - Ultralytics YOLOv8 Docs

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#4354] [Docs]
# Inference API - Ultralytics YOLOv8 Docs

Access object detection capabilities of YOLOv8 via our RESTful API. Learn how to use the YOLO Inference API with Python or CLI for swift object detection.

https://docs.ultralytics.com/hub/inference_api/

**Comment by IndShiv** [comment]
Hi when will this be available? Documentation says coming soon?

**Reply by glenn-jocher** [reply]
@IndShiv hello! The Inference API for YOLOv8 is currently in development, and we're working hard to make it available as soon as possible. Keep an eye on our documentation for updates, and we appreciate your patience and excitement for this feature! 😊🚀
